src/cheri_insts.sail

```diff
@@ -133,8 +134,12 @@ union clause ast = CJALR : (bits(12), regidx, regidx)
  *
  * An exception is raised if:
  *   - *cs1*.**tag** is not set.
- *   - *cs1* is sealed and is not a sentry.
- *   - *cs1* is a sentry and *imm* $\ne$ 0.
+ *   - *cs1* is sealed and *imm* $\ne$ 0
+ *   - Not all of the following combinations hold:
```

Comment:
Should this be if none of the following hold?

Reply:
Yes. But what I wrote is also true (distribution of not, or)

Reply:
Maybe I am screwing up the English language. Let me fix it

Reply:
Done